content_scripts/mode_normal.coffee

```diff
@@ -91,10 +91,19 @@ NormalModeCommands =
 
   copyCurrentUrl: ->
     chrome.runtime.sendMessage { handler: "getCurrentTabUrl" }, (url) ->
-      chrome.runtime.sendMessage { handler: "copyToClipboard", data: url }
+      HUD.copyToClipboard url
       url = url[0..25] + "...." if 28 < url.length
       HUD.showForDuration("Yanked #{url}", 2000)
 
+  openCopiedUrlInNewTab: (count) ->
+    HUD.pasteFromClipboard (url) ->
+      for i in [0...count] by 1
```

Comment:
I might change this to use `mkRepeatCommand` in `main.coffee` (to keep the `count` logic in as few places as possible).